app/forms/register_nonprofit_form/nonprofit_form.rb

```diff
@@ -0,0 +1,65 @@
+# License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later
+
+class RegisterNonprofitForm::NonprofitForm < ApplicationForm
+  attribute :name, :string, default: "" # needed because OnboardAccounts is goofy
+
+  attr_reader :website
+  attr_accessor :city,
+    :email,
+    :state_code,
+    :phone,
+    :zip_code
+
+  validates :zip_code, presence: true
+  validates :website, format: {with: URI.regexp}, if: proc { |form| form.website.present? }
+  # validates :email, format: {with:  URI::MailTo::EMAIL_REGEXP }, if: Proc.new { |form| form.email.present? }
+
+  def initialize(attributes = {})
+    super
+    @models = [
+      nonprofit
+    ]
+  end
+
+  def nonprofit
+    @nonprofit ||= Nonprofit.new(OnboardAccounts.set_nonprofit_defaults(
+      city: city,
+      email: email,
+      name: name,
+      phone: phone,
+      state_code: state_code,
+      website: website,
+      zip_code: zip_code
```

Comment:
Is it ruby 3 where we can start leaving off the variables in the hash param syntax? 

```
OnboardAccounts.set_nonprofit_defaults(city:, email:, name:, phone:, state_code:, website:, zip_code: ))
```

Reply:
3.1, I think.